supereeg/model.py

```diff
@@ -259,11 +273,12 @@ def set_locs(self, new_locs, include_original_locs=False):
 
 
     def predict(self, bo, nearest_neighbor=False, match_threshold='auto',
-                force_update=False, update_locs=True, preprocess='zscore'):
+                force_update=False, include_original_locs=True, preprocess='zscore'):
```

Comment:
What does `include_original_locs` do?  The name is confusing-- maybe it should be something like `force_include_bo_locs`?  Is the idea that this flag forces us to predict at the locations in the original brain object, even if they're not in the model?

Reply:
changed the name and yes the flag forces to predict at the locations in the original brain object if not in the model